leakcanary-android-core/src/main/java/leakcanary/internal/HeapDumpPolicy.kt

```diff
@@ -0,0 +1,28 @@
+package leakcanary.internal
+
+import leakcanary.LeakCanary
+import leakcanary.internal.HeapDumpPolicy.HeapDumpStatus.DISABLED_CONFIG_CHANGED
+import leakcanary.internal.HeapDumpPolicy.HeapDumpStatus.DISABLED_DEBUGGER_ATTACHED
+import leakcanary.internal.HeapDumpPolicy.HeapDumpStatus.ENABLED
+
+object HeapDumpPolicy {
+  enum class HeapDumpStatus {
+    ENABLED,
+    DISABLED_DEBUGGER_ATTACHED,
+    DISABLED_CONFIG_CHANGED
+  }
+
+  fun getHeapDumpStatus(): HeapDumpStatus {
```

Comment:
Can be converted to a Kotlin property

Reply:
To check whether tests were running, I had to pass in the resources object. So I can't use a Kotlin property anymore.